syllabus/module/2.7-slides.md

```diff
@@ -0,0 +1,399 @@
+---
```

Comment:
We only have 2.6 slides for this module, why there is 2.7? Please follow the plan: https://app.gitbook.com/o/sNkXo5TPjvamNNPMV8lh/s/sqkizIN3EVy0ti2xefEM/opencoursestack